.github/workflows/pr-run-checks.yml

```
@@ -0,0 +1,20 @@
+name: pr-run-checks
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+jobs:
+  check-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 15.x, 16.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: npm ci
+      - run: npm run build
```

If I'm reading this right, this job is to run the tests since the job name is `check-tests`.  Should this be change to this?

```suggestion
      - run: npm run test
```

---

Right so in this case no. Jest tests don't actually run off of the build artifacts, so technically, we should be testing for both `npm run build` _and_ `npm run test`. However, because we now have a post-build directive, tests get ran with the build command anyways.